### DISCUSSION TITLE
Feature request: live notification

### DISCUSSION THREAD

**Discussion by caiotizio** [FreeTubeApp/FreeTube#5352] [General]
hi there,
i always miss live streams from my subscribed channels, because i have no habit to switch into live tab. it is also inconvenient and boring to constantly switch to see if there is anything live.
I would like to have a notification (or whatever else) when some channel is live.
thanks for this great software.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @caiotizio this is a duplicate of https://github.com/FreeTubeApp/FreeTube/issues/4157 so i will be closing this. If you have implementation suggestions you can leave them in that issue.

> thanks for this great software.

You're welcome :)
